pkg/webhook/mutation.go

```diff
@@ -55,7 +55,7 @@ func init() {
 // TODO enable this once mutation is beta +kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch;update
 
 // AddMutatingWebhook registers the mutating webhook server with the manager
-func AddMutatingWebhook(mgr manager.Manager, client *opa.Client, processExcluder *process.Excluder) error {
+func AddMutatingWebhook(mgr manager.Manager, client *opa.Client, processExcluder *process.Excluder, mutationCache *mutation.System) error {
```

Comment:
are we storing the mutation system on the mutation webhook somewhere?

Reply:
I added the storing of the mutation system in mutationHandler.
The invocation of the mutate function will be added in a new PR. The code is ready in my private repo [(here)](https://github.com/mmirecki/gatekeeper/tree/frankenstein_image), but I'm waiting for some previous PR's to get merged to avoid rebasing on each change in previous PR's.

Reply:
+1

Reply:
I don't see the storage code here, but that's fine if it's meant to be in a later PR

Reply:
Forgot to push changes, should be better now